datafusion/common/Cargo.toml

```diff
@@ -34,6 +34,7 @@ path = "src/lib.rs"
 
 [features]
 avro = ["apache-avro"]
```

Comment:
As written, to be enabled, a user would have to use `datafusion_common` directly (they couldn't activate it through `datafusion`)

To enable it via datafusion we also need to add it to core/Cargo.toml, like this:

https://github.com/apache/arrow-datafusion/blob/cde74016e930ffd9c55eed403b84bcd026f38d0f/datafusion/core/Cargo.toml#L44

Reply:
Thanks I was little bit struggling on how it make on the datafusion level